src/scripts/main.js

```diff
@@ -1,3 +1,98 @@
 'use strict';
 
-// write code here
+const appendRowBtn = document.querySelector('.append-row');
+const removeRowBtn = document.querySelector('.remove-row');
+const appendColBtn = document.querySelector('.append-column');
+const removeColBtn = document.querySelector('.remove-column');
+const table = document.querySelector('.field');
+const container = document.querySelector('.container');
+let rowsCount = table.rows.length;
+let colsCount = table.rows[0].cells.length;
+
+const disableRowsBtn = (value) => {
+  switch (value) {
+    case 2:
+      removeRowBtn.setAttribute('disabled', true);
+
+      break;
+    case 10:
+      appendRowBtn.setAttribute('disabled', true);
+
+      break;
+    default:
+      removeRowBtn.removeAttribute('disabled');
+      appendRowBtn.removeAttribute('disabled');
```

Comment:
Better to add a break to all cases
<img width="754" alt="image" src="https://user-images.githubusercontent.com/99403198/214129286-638db5fe-2e03-4e02-bf94-e67196e60549.png">